app/templates/inbox/meetings/new.hbs

```diff
@@ -0,0 +1,70 @@
+{{!-- TODO: Uncomment this once the app uses `{{page-title}}` on all relevant pages --}}
+{{!-- {{page-title (t "inbox.meetings.new.title")}} --}}
+
+<AuModal
+  @modalTitle={{t "inbox.meetings.new.title"}}
+  @modalOpen={{true}}
+  @closeModal={{this.cancelMeetingCreation}}
+  as |Modal|
+>
+  <Modal.Body>
+    <form class="au-o-flow" id="create-meeting-form" {{on "submit" (perform this.saveMeetingTask)}}>
+      <div>
+        {{#let (unique-id) this.meeting.errors.bestuursorgaan as |id errors|}}
+          <AuLabel
+            @error={{if errors true false}}
+            for={{id}}
+          >
+            {{t "inbox.meetings.new.meeting.administrativeBody"}} <AuPill>Verplicht</AuPill>
+          </AuLabel>
+          <AdministrativeBodySelect
+            @id={{id}}
+            @selected={{this.meeting.bestuursorgaan}}
+            @onChange={{this.updateAdministrativeBody}}
+            @error={{if errors true false}}
+          />
+          {{#each errors as |error|}}
+            <AuHelpText @error={{true}}>
+              {{t error.message}}
+            </AuHelpText>
+          {{/each}}
+        {{/let}}
+      </div>
+      <div>
+        {{#let (unique-id) as |id|}}
+          <AuLabel for={{id}}>{{t "inbox.meetings.new.meeting.location"}}</AuLabel>
+          <AuInput @value={{this.meeting.opLocatie}} @width="block" id={{id}} type="text" />
+        {{/let}}
+      </div>
+      <div>
+        <AuLabel>{{t "inbox.meetings.new.meeting.plannedStart"}}</AuLabel>
+        <DateTimePicker @onChange={{fn (mut this.meeting.geplandeStart)}} @value={{this.meeting.geplandeStart}}/>
+      </div>
+      <div>
+        <AuLabel>{{t "inbox.meetings.new.meeting.start"}}</AuLabel>
```

Comment:
since we're improving the flow anyway and given that this is now separated I'd actually drop the start and end datetime pickers, as they only make sense after the meeting has happened

Reply:
Yea, I agree, that's one of the reasons I didn't create a single component that covers both cases. I had a gut feeling that they "looked the same, but aren't the same". I'll remove those fields.